src/web/mjs/connectors/AstralLibrary.mjs

```diff
@@ -10,5 +10,28 @@ export default class AstralLibrary extends WordPressMadaraNovel {
         this.url = 'https://astrallibrary.net';
 
         this.novelObstaclesQuery = 'div.ad';
+        this.queryMangas = 'div.item-summary';
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(new URL('page/'+page+1,'https://www.astrallibrary.net/manga-tag/manga/'));
```

Comment:
Use `this.url`, in case it changes it can be simply changed in a single place.
I think brackets are required to increment the page number, otherwise the `page` is first appended to the string and the `1` may also be appended, e.g. `'xxx' + 7 + 1` will result in `xxx71` which is probably not the desired outcome.

`new URL('/manga-tag/manga/page/' + (page + 1), this.url)`

Reply:
Obsolete since this method should not be overwritten